{
  return ctx.closures().callLocalSafe(new Callable<Object>(){
    @Nullable @Override public Object call() throws Exception {
      await();
      return null;
    }
  }
,true);
}
